sdk/data/aztables/sas_account.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 )
 
+// SASVersion is the default SAS Service Version
```

Comment:
SAS Version

Reply:
Comments are supposed to start with the name of the type they are documenting, so in this case leaving it all as one word.

Reply:
Why is this a variable, and exported?

Reply:
Sorry I was referring to the end of the comment.  "SASVersion is the default SAS version".  The extra "Service" seems redundant.

Reply:
@chlowell This is the default value used for `AccountSASSignatureValues`